doc/sphinx_source/modules/internals.rst

```diff
@@ -125,23 +146,65 @@ Triggering any Bind
+| MATCH_EXACT       | Check the triggering value exactly against the bind mask value (case insensitive)                                             |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_CASE        | Check the triggering value exactly against the bind mask value (case sensitive)                                               |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_MASK        | Check if the bind mask is contained within the triggering value, as a wildcarded value                                        |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_MODE        | Check if the triggering value is contained within the bind mask, as a wildcarded value                                        |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_CRON        | Check the triggering value against a bind mask formatted as a cron entry, ie "30 7 6 7 * " triggers a mask for "30 7 * * * "  |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_USE_ATTR     | Check the flags of the user match the flags required to trigger the bind                                                      |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_STACKABLE    | Allow one mask to be re-used to call multiple Tcl proc. Must be used with HT_STACKABLE_                                       |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_WANTRET      | With stacked binds, if the called Tcl proc called returns a '1', halt processing any further binds triggered by the action    |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_STACKRET     | Used with BIND_WANTRET; allow stacked binds to continue despite receiving a '1'                                               |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_ALTER_ARGS   | Replaces arguments (which ones?) with the result returned from the called Tcl proc                                            |
```

Comment:
I can't figure out what this was ever used for and it is not currently used for any bind types in Eggdrop. I propose leaving it undocumented.